services/ui-src/src/forms/mcpar/dpi/dpi.json

```diff
@@ -1,18 +1,181 @@
   },
   "form": {
     "id": "dpi",
     "fields": [
       {
         "id": "dpi-1",
-        "type": "text",
+        "type": "number",
+        "props": {
+          "label": "D1.X.1 Dedicated program integrity staff",
+          "hint": "Report or enter the number of dedicated program integrity staff for routine internal monitoring and compliance risks. Refer to 42 CFR 438.608(a)(1)(vii).",
+          "mask": "comma-separated"
+        }
+      },
+      {
+        "id": "dpi-2",
+        "type": "number",
+        "props": {
+          "label": "D1.X.2 Count of opened program integrity investigations",
+          "hint": "How many program integrity investigations have been opened by the plan in the past year?",
+          "mask": "comma-separated"
+        }
+      },
+      {
+        "id": "dpi-3",
+        "type": "number",
+        "props": {
+          "label": "D1.X.3 Ratio of opened program integrity investigations to enrollees",
+          "hint": "What is the ratio of program integrity investigations opened by the plan in the past year per 1,000 beneficiaries enrolled in the plan on the first day of the last month of the reporting year?",
+          "mask": "comma-separated"
+        }
+      },
+      {
+        "id": "dpi-4",
+        "type": "number",
+        "props": {
+          "label": "D1.X.4 Count of resolved program integrity investigations",
+          "hint": "How many program integrity investigations have been resolved by the plan in the past year?",
+          "mask": "comma-separated"
+        }
+      },
+      {
+        "id": "dpi-5",
+        "type": "number",
+        "props": {
+          "label": "D1.X.5 Ratio of resolved program integrity investigations to enrollees",
+          "hint": "What is the ratio of program integrity investigations resolved by the plan in the past year per 1,000 beneficiaries enrolled in the plan at the beginning of the reporting year?",
+          "mask": "comma-separated"
+        }
+      },
+      {
+        "id": "dpi-6",
+        "type": "radio",
+        "props": {
+          "label": "D1.X.6 Referral path for program integrity referrals to the state",
+          "hint": "What is the referral path that the plan uses to make program integrity referrals to the state? Select one.",
+          "choices": [
+            {
+              "name": "dpi-6-o1",
+              "type": "choice",
+              "label": "Makes referrals to the Medicaid Fraud Control Unit (MFCU) only",
+              "children": [
+                {
+                  "id": "dpi-7-o1",
+                  "type": "number",
+                  "props": {
+                    "label": "D1.X.7 Count of program integrity referrals to the state",
+                    "hint": "Enter the count of program integrity referrals that the plan made to the state in the past year. Enter the count of referrals made.",
+                    "mask": "comma-separated"
+                  }
+                }
```

Comment:
Well these are a super weird case, aren't they? After going back and forth a bunch, I think we should make the IDs for these 3 `dpi-7`s `dpi-7a`, `dpi-7b`, and `dpi-7c` to differentiate them from true options/choices and make it clear they are their own fields.